pkg/datastore/mysql/mysql.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2021 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"io/ioutil"
+
+	// mysql driver
+	_ "github.com/go-sql-driver/mysql"
+
+	"github.com/pipe-cd/pipe/pkg/datastore"
+	"go.uber.org/zap"
```

Comment:
nit: Move the importing of "datastore" into the last group.

Reply:
Sorry but my local lint stored imports in this order 😢 Is there anything missed on my local configuration? 🤔

Reply:
I think you should add an empty line between them (line 26, 27)/

Reply:
gotcha 👍 thank you 🙏 
```go
import (
	"context"
	"database/sql"
	"fmt"
	"io/ioutil"

	// mysql driver
	_ "github.com/go-sql-driver/mysql"
	"go.uber.org/zap"

	"github.com/pipe-cd/pipe/pkg/datastore"
)
```